### DISCUSSION TITLE
Cloning

### DISCUSSION THREAD

**Discussion by Bonk1971** [neonbjb/tortoise-tts#674] [General]
I have been using [tortoise-fast] for voice cloning but I have noticed that repo has pretty much stopped development due to the owner gettting hired by a competing company.  I have only recently come back here to see what is up and notice that @manmay-nakhashi has been very busy improving this repo. Thank you for your work on this Manmay! (this is crazy fast now.) Is there a way that I can use my custom checkpoints with the regular tortoise repo?  I am stumped as to how to do this?  I am not an experienced coder so maybe it is right in front of me?

**Comment by fakerybakery** [comment]
Should work as normal, just make sure it doesn't re-download the weights (try removing the hf_hub_download code once you've finished downloading the initial weights and hardcode the model paths in)
